src/OpenAuthenticode.Module/OpenAuthenticodeAzKey.cs

```diff
@@ -15,11 +15,14 @@ public sealed class GetOpenAuthenticodeAzKey : PSCmdlet
     [Alias("CertificateName")]
     public string Certificate { get; set; } = "";
 
+    [Parameter(Position = 2)]
```

Comment:
I think for optional values we should keep as non-positional in case we want to add something a bit more common in this position.

```
    [Parameter()]
```